pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
```

Comment:
Any reason why this was added?

Reply:
Is my second time with pnpm. 
Reading here https://github.com/pnpm/pnpm/releases/tag/v7.33.0 means that my installation of pnpm added these flag. I need to remove?

Reply:
I see, same happened for [version 8.x.x](https://github.com/pnpm/pnpm/releases/tag/v8.6.0)